src/MoBi.Presentation/Presenter/SelectContainerPresenter.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using MoBi.Assets;
+using MoBi.Core.Domain.Extensions;
+using MoBi.Core.Domain.Model;
+using MoBi.Presentation.DTO;
+using MoBi.Presentation.Mappers;
+using MoBi.Presentation.Views;
+using OSPSuite.Core.Domain;
+using OSPSuite.Presentation.Presenters;
+using OSPSuite.Utility.Extensions;
+
+namespace MoBi.Presentation.Presenter
+{
+   public interface ISelectContainerPresenter : ISelectObjectPathPresenter
+   {
+      ObjectPath Select();
+   }
+
+   public class SelectContainerPresenter : AbstractDisposablePresenter<ISelectObjectPathView, ISelectObjectPathPresenter>, ISelectContainerPresenter
```

Comment:
@rwmcintosh @Yuri05 I copied the code from the SelectEvent and made it so that we can select a container. I do not really understand the original code. Thsi one is pretty straightforward as i do not add any dummy container